test/check-machines-consoles

```diff
@@ -110,8 +110,6 @@ class TestMachinesConsoles(machineslib.VirtualMachinesCase):
         b.click("#SerialConsole button")
         b.wait_not_present("#pf-v5-c-console__type-selector + .pf-v5-c-select__menu")
 
-        b.wait_in_text(f"#{name}-terminal .xterm-accessibility-tree", f"Connected to domain '{name}'")
```

Comment:
That was strange indeed - this should rather check for something at the end of the boot sequence which doesn't scroll off, like the usual "login:" prompt?